docs/design/1533-host-specific-tasklist.md

```diff
@@ -222,20 +230,18 @@ This section illustrates the sequence diagrams for situations that may occur dur
 
 ![image alt text](1533-host-specific-tasklist-image_2.png)
 
-* When there are too many outstanding sessions and the user is trying to create a new one. The assumption here is that each activity worker is configured to have **MaxCurrentSessionExecutionSize = 1**. Host specific tasklist and worker is omitted in the diagram.
+* When there are too many outstanding sessions and the user is trying to create a new one. The assumption here is that each activity worker is configured to have **MaxCurrentSessionExecutionSize = 1**. Resource specific tasklist and session activity worker is omitted in the diagram.
```

Comment:
if resource specific tasklist and session activity worker are two separate things omitted from the diagram, then use "are omitted".